templates/conf.d/map/footer.erb

```diff
@@ -0,0 +1 @@
+}
```

Comment:
do we really need a template for this? You can directly set the `}` in the concat::fragment resource.